fuzzers/074-dump_all/ignored_wires/kintex7/xc7k160tffg676-2_ignored_wires.txt

```diff
@@ -0,0 +1,176 @@
+LIOI3_X0Y45/LIOI_I2GCLK_BOT1
```

Comment:
This file should be sorted, similarly as the other ones corresponding to the ignored_wires.

Reply:
OK done